ide/projectui/src/org/netbeans/modules/project/ui/ProjectUtilities.java

```diff
@@ -135,28 +130,25 @@ private void doClose(Project[] projects, boolean notifyUI, Wrapper wr) {
                 if (dobj != null) {
                     FileObject fobj = dobj.getPrimaryFile();
                     Project owner = ProjectConvertors.getNonConvertorOwner(fobj);
-                    ERR.log(Level.FINER, "Found {0} owned by {1} in {2} of {3}", new Object[] {fobj, owner, tc.getName(), tc.getClass()});
+                    LOG.log(Level.FINER, "Found {0} owned by {1} in {2} of {3}", new Object[] {fobj, owner, tc.getName(), tc.getClass()});
 
-                    if (listOfProjects.contains(owner)) {
+                    if (owner != null && listOfProjects.contains(owner)) {
                         if (notifyUI) {
                             openFiles.add(dobj);
                             tc2close.add(tc);
                         } else if (!dobj.isModified()) {
                             // when not called from UI, only include TCs that arenot modified
                             tc2close.add(tc);
                         }
-                        //#235897 split a single line to detect NPE better
```

Comment:
I guess you may have found the NPE they were search for 12 years ago in https://bz.apache.org/netbeans/show_bug.cgi?id=235897 ...

Reply:
yep I hope so. Although it might be still more to it.

Reply:
> the extra steps to locate NPEs are also no longer needed thanks to https://openjdk.org/jeps/358
Yep.